src/cairoWriter/writers/literalWriter.ts

```diff
@@ -31,16 +30,13 @@ export class LiteralWriter extends CairoASTNodeWriter {
         return [`0x${node.hexValue}`];
       }
       case LiteralKind.HexString:
-        switch (primitiveTypeToCairo(node.typeString)) {
-          case 'Uint256': {
-            return [
-              `Uint256(low=0x${node.hexValue.slice(32, 64)}, high=0x${node.hexValue.slice(0, 32)})`,
-            ];
-          }
-          case 'felt':
-            return [`0x${node.hexValue}`];
-          default:
-            throw new TranspileFailedError('Attempted to write unexpected cairo type');
+        if (
+          (primitiveTypeToCairo(node.typeString) as CairoPrimitiveIntType) ||
+          primitiveTypeToCairo(node.typeString) === 'felt'
+        ) {
+          return [`0x${node.hexValue}`];
+        } else {
+          throw new TranspileFailedError('Attempted to write unexpected cairo type');
```

Comment:
Let's make a function that throws  the error, then return:
```
validate(node);
return [`0x${node.hexValue`]
```
This way it can be reused for hexString and number